tests/integ/modin/frame/test_merge.py

```diff
@@ -226,27 +198,16 @@ def _verify_merge(
     indicator: Optional[Union[bool, str]] = False,
 ) -> None:
     """
-    For inner and outer join order of joined frame in Snowpark pandas is different from
-    native pandas. In Snowpark pandas we order by [left.row_position, right.row_position]
-    while in Native pandas output of joined frame is: first rows are grouped by the
-    keys, and order among keys is inherited from the left dataframe, and then the right
-    data frame.
-
-    In addition to above there are bugs in Native pandas where even left and right join
-    do not preserve order from left frame or right frame respectively.
+    To avoid comparison failure due to bugs in Native pandas we perform some custom
+    operation after merge.
+    Some example bugs:
+    left and right join do not preserve order from left frame or right frame respectively.
     https://github.com/pandas-dev/pandas/issues/40608
```

Comment:
but this has bug has been fixed now?

Reply:
Good catch, I should remove reference to this bug. In 2.x quite a few sorting related bugs are fixed so we no longer need custom sorting logic in this method.